entrypoint.sh

```diff
@@ -2,10 +2,12 @@
 
 docker --version >&2
 
-# add the node user to the docker group on the host
+# set the dockeronhost group id in the container to the group id from the host
 DOCKER_GROUP=$(stat -c '%g' /var/run/docker.sock)
-groupadd --non-unique --gid ${DOCKER_GROUP} dockeronhost
-usermod -aG dockeronhost node
+# try to modify an existing group id
+groupmod --non-unique --gid ${DOCKER_GROUP} dockeronhost
```

Comment:
Are you suggesting that the id of the docker group is changing? Or is this used for checking for an existing `dockeronhost` group?

This command will also emit a log message when it cannot find the group-id, so this does not improve the situation a lot?

``` 
$ docker run --rm -it node:10 bash
root@d40e014e9687:/# groupmod --non-unique --gid 999 dockeronhost
groupmod: group 'dockeronhost' does not exist
```